docs/content_management/taxonomy/taxonomy.md

```diff
@@ -28,16 +28,30 @@ The identifiers correspond to Field names defined in the Content Type. The `name
 ## Customize taxonomy structure
 
 You can create other taxonomies than the one predefined in the system, for example a Content category.
-To do it, first, create a new Content Type with `content_category` identifier and include the following Field Types:
 
-* `category_identifier` of `ezstring` type
-* `parent` of `ibexa_taxonomy_entry` type
-* `name` of `string` type
+To do it, first, create a new container to store the new taxonomy's items, for example a `folder` named Content categories.
```

Comment:
```suggestion
To do it, first, create a new container to store the new taxonomy's items, for example a folder named "Content categories".
```
Re: trick, I guess we need to make sure this is intended way to use taxonomy, ping @juskora

Reply:
I'm not sure, I think better idea would be to ask Maciej Kobus?